src/drem/etl/commercial.py

```diff
@@ -34,6 +34,11 @@
     )
 
     benchmarks = transform_benchmarks(benchmarks_dir)
-    vo_clean = transform_vo(vo_dirpath, benchmarks, benchmarks_dir / "Unmatched.txt")
+    vo_clean = transform_vo(
+        input_filepath=vo_dirpath,
+        data_dir=data_dir,
+        benchmarks_dir=benchmarks_dir,
+        external_dir=external_dir,
+    )
```

Comment:
```suggestion
    vo_clean = transform_vo()
```
Does this not need any inputs?